docs/clients/promtail/installation.md

```diff
@@ -1,41 +1,62 @@
-# Installation
-Promtail is distributed in binary and in container form.
+# Installing Promtail
 
-Once it is installed, you have basically two options for operating it:
-Either as a daemon sitting on every node, or as a sidecar for the application.
+Promtail is distributed as a binary, Docker container, and Helm chart.
```

Comment:
I would suggest to link the respective anchors `#binary`, `#docker` and `#helm`.